source/SuperTool/SuperTool.gpr.py

```diff
@@ -116,12 +126,15 @@
     name=_("Generic citation filter"),
     description=_("Generic citation filter"),
     version=VERSION,
+    audience = EVERYONE,
     authors=["Kari Kujansuu"],
     authors_email=["kari.kujansuu@gmail.com"],
     gramps_target_version=major_version,
     status=STABLE,
+    audience = EVERYONE,
```

Comment:
Added twice?

Reply:
No, it should have only been once.

Darn! Thought I'd caught all those! It was late and I was tired. 

He had a few with author entries. (Some with "kku")

Some have the licensing header too.